tests/phpunit/integration/tests/Admin/Cross_Origin_Isolation.php

```diff
@@ -87,9 +92,8 @@ public function test_register() {
 	public function test_is_needed() {
 		wp_set_current_user( self::$admin_id );
 		update_user_meta( self::$admin_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY, true );
-		$instance = new \Google\Web_Stories\Admin\Cross_Origin_Isolation();
-		$result   = $this->call_private_method( $instance, 'is_needed' );
-		$this->assertTrue( $result );
+
+		$this->assertTrue( \Google\Web_Stories\Admin\Cross_Origin_Isolation::is_needed() );
```

Comment:
Why can you do this?
```suggestion
		$this->assertTrue( $this->instance::is_needed() );
```

Reply:
Because it's a static method and should be called as such ideally.